src/components/manage/Contents/Contents.jsx

```diff
@@ -1210,17 +1258,18 @@ class Contents extends Component {
                             <Icon
                               name={
                                 this.state.selected.length === 0
-                                  ? 'square outline'
```

Comment:
I would do it with full component, it's more readable, so:

condition ? <Icon name={foo} ... /> : <Icon name={bar} ... />

Reply:
Ok!! I will do that :)